test/parsers/aircraftCommandParser/argumentValidator.spec.js

```diff
@@ -136,7 +137,7 @@ ava('.altitudeValidator() returns a string when passed anything other than exped
 });
 
 ava('.optionalAltitudeValidator() returns undefined when no value is passed', t => {
-    let result = optionalAltitudeValidator([]);
+    const result = optionalAltitudeValidator([]);
     t.true(typeof result === 'undefined');
```

Comment:
missing empty line after `const`

Reply:
Technically I agree with you but the rest of the file follows the same pattern. I would like to keep the file consistent so we either stick with the current pattern or change the whole file. Let me know which way you prefer.

Reply:
Works for me, let’s leave it!